README.md

```diff
@@ -569,6 +570,91 @@ Motion Command | Description
 `<operator>z<char><char>`|Perform `<operator>` forward to the first occurence of `<char><char>`
 `<operator>Z<char><char>`|Perform `<operator>` backward to the first occurence of `<char><char>`
 
+### Input Method
+(Support macOS and windows for now, linux will come soon)
+
```

Comment:
Can you add a small section here saying something along the lines of 'there are two ways you can configure input method, using `im-select` (with a link to your GH), or your own program'